src/script/conversation/ConversationAccessPermission.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CONVERSATION_ACCESS, ACCESS_ROLE_V2} from '@wireapp/api-client/src/conversation/';
+import {combinePermissions, hasPermissions} from '../user/UserPermission';
+import {ACCESS_STATE, TEAM} from './AccessState';
+
+export const ACCESS_TYPES = {
+  GUEST: 1 << 0,
+  NON_TEAM_MEMBER: 1 << 1,
+  SERVICE: 1 << 3,
+  TEAM_MEMBER: 1 << 2,
+};
```

Comment:
We should probably disable the automatic sorting from eslint here. 
I think it doesn't make sense for those kind of structure

Reply:
id turn it off in the other bitmap file also, I noticed it there